nuxeo-importer-core/src/main/java/org/nuxeo/ecm/platform/importer/service/ImporterConfigurationDescriptor.java

```diff
@@ -42,6 +42,9 @@
     @XNode("repository")
     protected String repository;
 
+    @XNode("enablePerfLogging")
+    protected Boolean enablePerfLogging;
+
```

Comment:
would be nice to add the since annotation here, for documentation of the extension point configuration

Reply:
actually would be nice to add this to the extension point documentation too